### ISSUE TITLE
"File changed" dialog should include filename 

### ISSUE THREAD

**Issue by ivanov** [jupyterlab/jupyterlab#6229]
**Describe the bug**

The current dialog for File Changed should be improved to include the actual filename in question. Currently, this reads:

> File Changed
> The file has changed on disk since the last time it was opened or saved. Do you want to overwrite the file on disk with the version open here, or load the version on disk (revert)?
> 

**To Reproduce**

1. Create a file (say `hi.txt`) and open it in JupyterLab text editor
2. Modify the file outside of the text editor, say by running the following code:

```python
with open("hi.txt", "a") as f:
    f.write("it's more fun to compute")
```
3.  Hit "Save" back in the JupyterLab text editor instance opened to "hi.txt"
4. This will trigger the "File Changed" dialog

![image](https://user-images.githubusercontent.com/118211/56532339-71830100-650a-11e9-975a-02cee7c89dbe.png)

**Expected behavior**
We should include the name of the file that is causing the dialog to trigger.

**Comment by ivanov** [comment]
Not this is not exclusive to the text editor, you can trigger the same dialog for other file types (notebooks, etc)

**Comment by jasongrout** [comment]
The code for this message is at https://github.com/jupyterlab/jupyterlab/blob/cf1063590f3f438b771b679fafc2eeed0838421e/packages/docregistry/src/context.ts#L698-L701

**Comment by bsyouness** [comment]
Working on this!

**Comment by blink1073** [comment]
Fixed by #6229.
